diff --git a/node_modules/metro-config/src/loadConfig.js b/node_modules/metro-config/src/loadConfig.js
index 7ac9d88..1d818e2 100644
--- a/node_modules/metro-config/src/loadConfig.js
+++ b/node_modules/metro-config/src/loadConfig.js
@@ -15,6 +15,7 @@ var MetroCache = _interopRequireWildcard(require("metro-cache"));
 var _os = require("os");
 var path = _interopRequireWildcard(require("path"));
 var _yaml = require("yaml");
+var _url = require("url");
 function _getRequireWildcardCache(e) {
   if ("function" != typeof WeakMap) return null;
   var r = new WeakMap(),
@@ -81,9 +82,9 @@ async function resolveConfig(filePath, cwd) {
     filePath != null
       ? resolve(filePath)
       : searchForConfigFile(
-          path.resolve(process.cwd(), cwd ?? ""),
-          (0, _os.homedir)(),
-        );
+        path.resolve(process.cwd(), cwd ?? ""),
+        (0, _os.homedir)(),
+      );
   if (configPath == null) {
     return {
       isEmpty: true,
@@ -99,28 +100,28 @@ function mergeConfigObjects(base, overrides) {
     ...overrides,
     ...(overrides.cacheStores != null
       ? {
-          cacheStores:
-            typeof overrides.cacheStores === "function"
-              ? overrides.cacheStores(MetroCache)
-              : overrides.cacheStores,
-        }
+        cacheStores:
+          typeof overrides.cacheStores === "function"
+            ? overrides.cacheStores(MetroCache)
+            : overrides.cacheStores,
+      }
       : null),
     resolver: {
       ...base.resolver,
       ...overrides.resolver,
       ...(overrides.resolver?.dependencyExtractor != null
         ? {
-            dependencyExtractor: resolve(
-              overrides.resolver.dependencyExtractor,
-            ),
-          }
+          dependencyExtractor: resolve(
+            overrides.resolver.dependencyExtractor,
+          ),
+        }
         : null),
       ...(overrides.resolver?.hasteImplModulePath != null
         ? {
-            hasteImplModulePath: resolve(
-              overrides.resolver.hasteImplModulePath,
-            ),
-          }
+          hasteImplModulePath: resolve(
+            overrides.resolver.hasteImplModulePath,
+          ),
+        }
         : null),
     },
     serializer: {
@@ -132,10 +133,10 @@ function mergeConfigObjects(base, overrides) {
       ...overrides.transformer,
       ...(overrides.transformer?.babelTransformerPath != null
         ? {
-            babelTransformerPath: resolve(
-              overrides.transformer.babelTransformerPath,
-            ),
-          }
+          babelTransformerPath: resolve(
+            overrides.transformer.babelTransformerPath,
+          ),
+        }
         : null),
     },
     server: {
@@ -245,7 +246,7 @@ function overrideConfigWithArguments(config, argv) {
   }
   if (argv.verbose === false) {
     output.reporter = {
-      update: () => {},
+      update: () => { },
     };
   }
   return mergeConfig(config, output);
@@ -289,7 +290,11 @@ async function loadConfigFile(absolutePath) {
       }
     } catch (e) {
       try {
-        const configModule = await import(absolutePath);
+        const importPath =
+          process.platform === "win32"
+            ? (0, _url.pathToFileURL)(absolutePath).href
+            : absolutePath;
+        const configModule = await import(importPath);
         config = await configModule.default;
       } catch (error) {
         let prefix = `Error loading Metro config at: ${absolutePath}\n`;
@@ -313,7 +318,7 @@ async function loadConfigFile(absolutePath) {
   } else {
     throw new Error(
       `Unsupported config file extension: ${extension}. ` +
-        `Supported extensions are ${[...JS_EXTENSIONS, ...TS_EXTENSIONS, ...YAML_EXTENSIONS].map((ext) => (ext === "" ? "none" : `${ext}`)).join()})}.`,
+      `Supported extensions are ${[...JS_EXTENSIONS, ...TS_EXTENSIONS, ...YAML_EXTENSIONS].map((ext) => (ext === "" ? "none" : `${ext}`)).join()})}.`,
     );
   }
   return {
